src/main/webapp/app/shared/http/file-uploader.service.ts

```diff
@@ -49,6 +48,41 @@ export class FileUploaderService {
         return this.http.post<FileUploadResponse>(url, formData).toPromise();
     }
 
+    /**
+     * Function which uploads a file. It checks for supported file extensions and file size.
+     * Options must be passed as a dictionary. E.g: { keepFileName: true }
```

Comment:
Maybe you could define a type for the options instead of having any here?

Reply:
Otherwise another programmer would not know easily what options are available

Reply:
Good point, I added that now.